views/vote/citizen.pug

```diff
@@ -0,0 +1,20 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = __('Register to vote - citizenship')
+  -var citizen = !hasData(data, 'vote.citizen') ? '' : data.vote.citizenship
```

Comment:
Ah ha, pretty sure this should be "data.vote.citizenship".

Reply:
Actually the second part should be data.vote.citizen! Originally I started with a mix of citizenship and citizen, and eventually picked one to unify. This must be a leftover.